sim/dex.ts

```diff
@@ -826,6 +826,22 @@ export class ModdedDex {
 				ruleTable.set(ruleSpec, '');
 				continue;
 			}
+			if (ruleSpec.split(":").length > 1) {
+				const parts = ruleSpec.split(":");
+				if (parts[0] === 'teamlength') {
+					const vals = ruleSpec.split(":")[1].split('-');
+					const onValidate = parseInt(vals[1]);
+					const onBattle = parseInt(vals[0]);
+					format.teamLength = {
+						validate: [onBattle, onValidate],
+						battle: onBattle
+					}
+				}
+				else if (parts[0] === 'level') {
```

Comment:
Ugh I hate that tslint doesn't have whitespace validation. PS requires `} else ...`

Reply:
Oh, wait, Travis _did_ complain about it. I take back my complaints.

Reply:
Yea Travis complained and I fixed